ldon/rustfmt.toml

```diff
@@ -0,0 +1,98 @@
+# title: Yatima Standard Rustfmt Configuration
+# author: John C. Burnham <john@yatima.io>
+
+# For additional information about Rustfmt options see:
+# https://rust-lang.github.io/rustfmt/?version=v1.5.1&
+
+# General
+version              = "One"              # rustfmt 2.0 is not released yet
+edition              = "2021"             # Rust edition
+unstable_features    = false              # don't use unstable rustfmt features
+newline_style        = "Unix"             # Never allow Windows' \n\r newlines
+# disable_all_formatting = false            # mostly useless setting
+
+# Indent
+hard_tabs            = false              # hard tabs break consistent layout
+tab_spaces           = 2                  # override default 4 space indent
```

Comment:
I'm personally pro tabs-into-2-spaces, but this may prove a contentious change from the current Rust-standard 4 spaces.

Reply:
I'm fine with being contentious, and I feel strongly that 2 spaces is the better default